obspy/io/ascii/core.py

```diff
@@ -44,7 +44,7 @@
 from obspy.core.util import AttribDict, loadtxt
 
 
-HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %d sps, %.26s, %s, %s, %s\n"
+HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %s sps, %.26s, %s, %s, %s\n"
 
```

Comment:
> how about:
> 
> In [18]: ('%s' % 12.0000000001).rstrip('0').rstrip('.')
> Out[18]: '12.0000000001'
> 
> In [19]: ('%s' % 12.0000000000).rstrip('0').rstrip('.')
> Out[19]: '12'

I've implemented something along that line (this was not safe.. consider `1200` -> `'12'`). I would have been OK with just plain `%s`, hope we didn't miss an edge case were the stripping is doing harm.